exporter/opentelemetry-exporter-otlp-proto-http/pyproject.toml

```diff
@@ -24,8 +24,7 @@ classifiers = [
   "Programming Language :: Python :: 3.10",
 ]
 dependencies = [
-  "backoff >= 1.10.0, < 2.0.0; python_version<'3.7'",
-  "backoff >= 1.10.0, < 3.0.0; python_version>='3.7'",
+  "backoff ~= 2.0",
```

Comment:
Updating backoff to 2.0 is not necessary for this solution. Thoughts on relaxing this?